spotim-comments.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /**
- * Plugin Name:         Spot.IM Comments
+ * Plugin Name:         OpenWeb.Com Comments
  * Plugin URI:          https://wordpress.org/plugins/spotim-comments/
  * Description:         Real-time comments widget turns your site into its own content-circulating ecosystem.
  * Version:             4.5.2
- * Author:              Spot.IM
+ * Author:              OpenWeb.Com
  * Author URI:          https://github.com/SpotIM
  * License:             GPLv2
  * License URI:         license.txt
- * Text Domain:         spotim-comments
+ * Text Domain:         ow

```

Comment:
Are you sure about this? What's the plugin folder name? How about `openweb-comments` or `openweb`?

Reply:
@chandrapatel 
>Are you sure about this?

I and @ItsMePN decided to use `ow` (stand for OpenWeb) as text-domain and package name.

>What's the plugin folder name?

We are keeping same plugin slug i.e. [spotim-comments](https://wordpress.org/plugins/spotim-comments/) so I think folder name will be `spotim-comments`. OR is it possible to rename folder name to `openweb-comments` ?

>How about openweb-comments or openweb?

I am ok with that but @ItsMePN can you please confirm on this?

Reply:
@chandrapatel - please let us know if you see any issues if we continue with `ow` because Abhijit has started with that already. I recall the translations are somehow dependent on directory name + text domain. So how can we deal with that hear because I guess we can not change the base directory name.

Reply:
>is it possible to rename folder name to openweb-comments ?

~We can't rename the folder. Yes, we can keep `ow` but I felt it's too short. We mentioned `Text Domain:` in plugin header so it's fine.~

~@abhijitrakas Please make sure the languages file of this plugin loads correctly when we change the text-domain.~

Wait, I think we can't change the text domain if this plugin already being translated using the old text-domain. Because it can causes issues with already translated(languages) files on WP.org